gh_release/ECHAM5_model.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0"?>
+<metadata>
+  <idinfo/>
+  <citation>
+    <citeinfo>
+      <origin>Alison P Appling</origin>
```

Comment:
and actually, if I had re-rendered, this would have the actual author list (the yaml is further along)